include/fmt/core.h

```diff
@@ -1500,19 +1616,52 @@ class dynamic_format_arg_store
     if (internal::const_check(need_copy<T>::value))
       emplace_arg(dynamic_args_.push<stored_type<T>>(arg));
     else
-      emplace_arg(arg);
+      emplace_arg(static_cast<const internal::unwrap_reference_t<T>&>(arg));
   }
 
   /**
+    \rst
     Adds a reference to the argument into the dynamic store for later passing to
-    a formating function.
+    a formating function. Supports wrapped named arguments.
```

Comment:
I'm not sure users will know what "wrapped" means in this context. How about the following?

>Supports named arguments wrapped in `std::reference_wrapper`.

Reply:
Ok